Microsoft.Toolkit.Uwp.UI/Triggers/NetworkConnectionStateTrigger.cs

```diff
@@ -22,13 +25,14 @@ public class NetworkConnectionStateTrigger : StateTriggerBase
         public NetworkConnectionStateTrigger()
         {
             _dispatcherQueue = DispatcherQueue.GetForCurrentThread();
-            var weakEvent =
-                new WeakEventListener<NetworkConnectionStateTrigger, object>(this)
-                {
-                    OnEventAction = (instance, source) => NetworkInformation_NetworkStatusChanged(source),
-                    OnDetachAction = (weakEventListener) => NetworkInformation.NetworkStatusChanged -= weakEventListener.OnEvent
-                };
-            NetworkInformation.NetworkStatusChanged += weakEvent.OnEvent;
+
+            _weakEvent = new WeakEventListener<NetworkConnectionStateTrigger, object, EventArgs>(this)
+            {
+                OnEventAction = (instance, source, eventArgs) => { NetworkInformation_NetworkStatusChanged(source); },
```

Comment:
As per https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/issues/262, we should do something like when calling the function in the event handler:
```
                OnEventAction = static (instance, source, eventArgs) => { instance.NetworkInformation_NetworkStatusChanged(source); },
```

Reply:
I've updated the file to reflect this change.